ivy_tests/test_ivy/helpers/hypothesis_helpers/general_helpers.py

```diff
@@ -549,4 +547,11 @@ def embedding_helper(draw):
         ).filter(lambda x: x[1][0].shape[-1] == embedding_dim)
     )
     padding_idx = draw(st.integers(min_value=0, max_value=num_embeddings - 1))
-    return dtype_indices + dtype_weight, indices[0], weight[0], padding_idx
+    return (
+        dtype_indices + dtype_weight,
+        indices[0],
+        weight[0],
+        padding_idx,
+        num_embeddings,
+        embedding_dim,
+    )
```

Comment:
Let's make sure the new returns doesn't break old usage of the embedding helper.